packages/remix-dev/vite/static/refresh-utils.cjs

```diff
@@ -13,7 +13,7 @@ function debounce(fn, delay) {
 const enqueueUpdate = debounce(async () => {
   let manifest;
   if (routeUpdates.size > 0) {
-    manifest = JSON.parse(JSON.stringify(__remixManifest));
+    manifest = structuredClone(__remixManifest);
```

Comment:
I think without feature detection, this would technically be a breaking change for some end users because of the browser support.  Remix uses [`<script type=module>`](https://caniuse.com/?search=type%3D%22module%22) so we have a loose baseline of browser support back to Chrome 61.  But then due to PE, we can also support versions prior to 61 because browsers won't understand `<script type="module">` and ignore it.

`structuredClone` is not available until Chrome 98 so this PR would break existing Remix users on Chrome 61 through 97 (anyone on Chrome <61 wouldn't have HMR since they wouldn't have scripts loaded).  

```suggestion
    manifest = typeof structuredClone === 'function' ? 
      structuredClone(__remixManifest) : 
      JSON.parse(JSON.stringify(__remixManifest));
```